src/Components/AppConfig/AppConfig.tsx

```diff
@@ -105,6 +116,34 @@ const AppConfig = ({ plugin }: Props) => {
           />
         </Field>
 
+        <Field
+          className={styles.marginTop}
+          description={
+            <span>
+              Disables Logs Drilldown&apos;s usage of the{' '}
+              <a
+                className="external-link"
+                href="https://grafana.com/docs/loki/latest/reference/loki-http-api/#patterns-detection"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Loki Patterns API
+              </a>{' '}
```

Comment:
Does this actually turn it off in the loki config? https://grafana.com/docs/loki/latest/reference/loki-http-api/#patterns-detection. I thought this was just shutting it down on the ui side.

Reply:
no we can't (and shouldn't) alter the Loki config from Grafana, it just stops us from hitting the API, so if folks don't have it configured they won't see the warning toast on every route change